### PR TITLE
fix(examples): include opacity on utils and settings

### PR DESCRIPTION
**Pull request by PablitoGS**

### CODE REVIEW

File: src/_settings.scss [hunk 1]
```diff
@@ -20,8 +20,9 @@
 // URLs
 $url-statics: '' !default;
 
+@import './settings/box-style';
```
Comment (julian-gernun) [inline src/_settings.scss]: Why did you have to move this?

File: src/utils.scss [hunk 2]
```diff
@@ -1,9 +1,10 @@
 // Utils
 @import './utils/breakpoints';
+@import './utils/colors';
```
Comment (julian-gernun) [inline src/utils.scss]: Why did you moved this?

Comment (PablitoGS) [reply src/utils.scss]: I justmoved it to have an alfalbetic order. It may be easier to find a partial scss

File: src/_settings.scss (hunk 1 above)
Comment (PablitoGS) [reply src/_settings.scss]: I justmoved it to have an alfalbetic order. It may be easier to find a partial scss

**Pull request merged**